app/api/routes/login.py

```diff
@@ -1,8 +1,31 @@
-from fastapi import APIRouter
+from fastapi import APIRouter, Depends, HTTPException, status
+from fastapi.security import OAuth2PasswordBearer, OAuth2PasswordRequestForm
+from models import Token, pwd_context, create_access_token
+from user import users_db
+from datetime import timedelta
 
 router = APIRouter()
 
-# make sure to remove this function
-@router.get("/example")
-def example():
-    return {"message": "this is an example"}
+oauth2_scheme = OAuth2PasswordBearer(tokenUrl="login")
```

Comment:
I saw this line many times. please don't repeat the code and try to find a way to stop that.

Reply:
This line "oauth2_scheme = OAuth2PasswordBearer(tokenUrl="login")
" you mean ?

Reply:
Yes, I saw it twice

Reply:
would you please see if the last edition meet what you mean?

Reply:
I think it's ok now, thanks